internal/timeperiod/timeperiod.go

```diff
@@ -77,12 +111,16 @@ func (e *Entry) TableName() string {
 // MarshalLogObject implements the zapcore.ObjectMarshaler interface.
 func (e *Entry) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
 	encoder.AddInt64("id", e.ID)
+	encoder.AddInt64("timeperiod-id", e.TimePeriodID)
 	encoder.AddTime("start", e.StartTime.Time())
 	encoder.AddTime("end", e.EndTime.Time())
 	encoder.AddString("timezone", e.Timezone)
 	if e.RRule.Valid {
 		encoder.AddString("rrule", e.RRule.String)
 	}
+	if e.RotationMemberID.Valid {
+		encoder.AddInt64("rotation-member-id", e.RotationMemberID.Int64)
```

Comment:
There's both in the code base, but `_` are more prominent.

```
$ grep -Er '(zap|encoder\.Add).*".*_.*"' internal | wc -l
27
$ grep -Er '(zap|encoder\.Add).*".*-.*"' internal | wc -l
17
```

Thus, I am going to change it to `_` and maybe this could be unified later on.

Reply:
New changes introduced with this PR! :)
```bash
$ grep -Enr '(zap\.|encoder\.).*\(".*-.*"' internal
internal/config/incremental_sync.go:110:                        zap.Int("deleted-elements", countDel),
internal/config/incremental_sync.go:111:                        zap.Int("faulty-elements", countErr),
internal/config/incremental_sync.go:112:                        zap.Int("loaded-elements", countLoad))
internal/config/incremental_sync.go:241:                        zap.Int("faulty-elements", countErr),
internal/config/incremental_sync.go:242:                        zap.Int("deleted-unknown-elements", countDelSkip),
internal/config/incremental_sync.go:243:                        zap.Int("deleted-elements", countDel),
internal/config/incremental_sync.go:244:                        zap.Int("updated-elements", countUpdate),
internal/config/incremental_sync.go:245:                        zap.Int("new-elements", countNew))
```